web/models/run.ex

```diff
@@ -5,13 +5,15 @@ defmodule Gatherto.Run do
     field :title, :string
     field :description, :string
     field :time, Ecto.DateTime
+    field :minimum_distance, :integer
+    field :maximum_distance, :integer
     belongs_to :created_by, Gatherto.Athlete, foreign_key: :athlete_id
     many_to_many :athletes, Gatherto.Athlete, join_through: Gatherto.AthleteRun
 
     timestamps()
   end
 
-  @required_params [:title, :description, :time]
+  @required_params [:title, :description, :time, :minimum_distance, :maximum_distance]
 
```

Comment:
That sounds like a good idea to me. I'll defer to your experience with running groups and make those changes to the request. Thanks for the feedback.